eqasc/code/scripts/clf.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
```

Comment:
Are these scripts needed? They aren't used by the evaluator...

Reply:
We can remove entire scripts/ folder as none of it is used by evaluator

Reply:
👍 

Thanks! I'll remove it.